src/app/globals.css

```
@@ -2,6 +2,17 @@
 @import "tw-animate-css";
 @import "../shared/ui/tokens.css";
 
+@keyframes slideInUp {
```

Stylelint (17.13.0)

[error] Expected keyframe name "slideInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)